libcontainer/nsenter/nsexec.c

```diff
@@ -1053,6 +1083,14 @@ void nsexec(void)
 					bail("failed to sync with parent: SYNC_MOUNTSOURCES_ACK: got %u", s);
 			}
 
+			/*
+			 * Update timens offsets
+			 * set boottime and monotonic offsets
```

Comment:
nit: missing periods. Please write complete sentences in comments, otherwise it's harder to read (and maintain).